gcc/rust/typecheck/rust-hir-type-check.h

```diff
@@ -24,6 +24,7 @@
 #include "rust-hir-trait-reference.h"
 #include "rust-autoderef.h"
 #include "rust-tyty-region.h"
+#include "rust-tyty-variance-analysis.h"
 
 #include <stack>
```

Comment:
just noticed this, from d42498b69e27. I had missed it! we should not be including system headers like this directly, we have to be careful with this. I can open an issue but you don't have to be the one to fix it if you don't want to

Reply:
#2920

Reply:
It will be easier to fix in next PR, than to do all the rebasing and testing. Especially since you already made the issue.